src/lib/helpers/downloadFile.ts

```diff
@@ -134,14 +134,14 @@ const downloadAndDecompress = async (
   const dirPath = await getDownloadDirPath(get, pathStart);
 
   try {
-    await fsPromises.access(dirPath);
+    await access(dirPath);
     return dirPath;
   } catch {}
 
   const archivePath = join(dirPath, ARCHIVE_FILE);
   await downloadFile(archivePath, get);
   await decompress(archivePath, dirPath);
-  await fsPromises.unlink(archivePath);
+  await rm(archivePath, { force: true, recursive: true });
```

Comment:
Why do you remove archive with `--rf`? It's super risky: a typo in a path, and you'll delete a whole directory.

Since you are removing a file, you do not need `recursive: true`.

Reply:
agreed, no need for {recursive: true} here. will fix it. But operation is still safe cause all of this happens inside .fluence dir anyway